tree/src/main/java/org/enginehub/linbus/tree/LinListTag.java

```diff
@@ -219,6 +220,19 @@ public T get(int index) {
         return value.get(index);
     }
 
+    /**
+     * Transform the tag at the given index.
+     *
+     * @param index the index of the tag to transform
+     * @param transformer the function to transform the tag
+     * @return the new list tag
+     */
+    public LinListTag<T> transformTag(int index, Function<T, T> transformer) {
```

Comment:
Could theoretically use a `UnaryOperator<T>` instead of the `Function<T, T>` - but that's only a design-choice